domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/PartAdmissionResponse.java

```diff
@@ -0,0 +1,110 @@
+package uk.gov.hmcts.cmc.domain.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+import org.hibernate.validator.constraints.NotBlank;
+import uk.gov.hmcts.cmc.domain.models.legalrep.StatementOfTruth;
+import uk.gov.hmcts.cmc.domain.models.party.Party;
+import uk.gov.hmcts.cmc.domain.models.response.DefendantPaymentPlan;
+import uk.gov.hmcts.cmc.domain.models.response.Evidence;
+import uk.gov.hmcts.cmc.domain.models.response.HowMuchOwed;
+import uk.gov.hmcts.cmc.domain.models.response.Timeline;
+import java.util.Objects;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_ABSENT)
+public class PartAdmissionResponse extends Response {
+
+    public enum PartAdmissionType {
+        AMOUNT_TOO_HIGH,
+        PAID_WHAT_BELIEVED_WAS_OWED
+    }
+
+    @NotNull
+    private final PartAdmissionType partAdmissionType;
+
+    @Valid
+    private final Evidence evidence;
+
+    @Valid
+    @NotNull
+    private final HowMuchOwed howMuchOwed;
+
+    @Valid
+    private final Timeline timeline;
+
+    @Valid
+    private final DefendantPaymentPlan defendantPaymentPlan;
+
+    @NotBlank
+    @Size(max = 99000)
+    private final String impactOfDispute;
+
+    public PartAdmissionType getPartialAdmissionType() {
+        return partAdmissionType;
+    }
+
+
+    public PartAdmissionResponse(
+        final FreeMediationOption freeMediation,
+        final MoreTimeNeededOption moreTimeNeeded,
+        final Party defendant,
+        final StatementOfTruth statementOfTruth,
+        final Evidence evidence,
+        final HowMuchOwed howMuchOwed,
+        final Timeline timeline,
+        final DefendantPaymentPlan defendantPaymentPlan,
+        final String impactOfDispute,
+        final PartAdmissionType partAdmissionType) {
```

Comment:
I'd prefer if the `) {` was on a separate line, to help visually separate argument list from constructor body